### DISCUSSION TITLE
termux for android 4.4.2

### DISCUSSION THREAD

**Discussion by arshamX** [termux/termux-app#3260] [Q&A] [answered]
which version of termux should I use on 4.4.2 android phone

**Comment by agnostic-apollo** [comment, marked as answer]
Termux never supported `4.4`. Only `>= 5.0` is supported.

**Comment by amysteriousdude** [comment]
> which version of termux should I use on 4.4.2 android phone

You can use like LinuxDeploy if you have root enabled.
